### PR TITLE
Workshop foo

### PR DESCRIPTION
**Pull request by ctb**

### CODE REVIEW

File: src/2015-3hr-remote-workshops.rst [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+* We limited ourselves to four rooms total, which decreased the technical
+  challenges.
+
+Some things could be improved or regularized --
+
+* I think we should have the presenter check in with each room at each
+  challenge/stopping point. Sometimes she went a bit fast for us.
+  Alternatively, someone should create a virtual sticky site where we
+  can use stickies remotely.
```
Comment (rgaiacs) [inline src/2015-3hr-remote-workshops.rst]: :sparkling_heart:

**Comment by apawlik** [comment]
Thanks for this @ctb!  Really interesting and informative. As to practicalities, did the class sites have any live feedback channel? That is, hangouts on air is a broadcasting tool. I know that @gvwilson run remote workshops via two way Hangouts and says that it was important that classes had good microphones so that the Instructor could hear the feedback and questions. 

I'm wondering if there are possibly two options here :
1) have good support locally from TAs in which case the questions and feedback can be via text (Etherpad or something) 
2) there is little TA support available so it would need to be two way communication (regular hangout rather than on air) 

The disadvantage of the latter is that things don't get recorded. However, I've been thinking about the remote workshops also as a way to deliver SWC in places where they may struggle to get funds to cover instructor expenses (let alone the fee). It would be great to increase out outreach. Though in these cases, the local helpers may be possibly in low numbers and also will be unfamiliar with the method of teaching. 

I really like the idea of using this as a way of teaching more advanced stuff. 
The benefits for instructors are also well pointed out. 

Really great to see this going!

**Comment by ctb** [comment]
On Sun, Dec 06, 2015 at 03:50:33AM -0800, Aleksandra Pawlik wrote:
> Thanks for this @ctb!  Really interesting and informative. As to practicalities, did the class sites have any live feedback channel? That is, hangouts on air is a broadcasting tool. I know that @gvwilson run remote workshops via two way Hangouts and says that it was important that classes had good microphones so that the Instructor could hear the feedback and questions. 

Clarified in the post --

* the four rooms participated in the hangout;

* we had a gitter side channel;

* the youtube bit was solely for recording and viewing by people elsewhere

> I'm wondering if there are possibly two options here :
> 1) have good support locally from TAs in which case the questions and feedback can be via text (Etherpad or something) 

exactly :)

> 2) there is little TA support available so it would need to be two way communication (regular hangout rather than on air) 
> 
> The disadvantage of the latter is that things don't get recorded. However, I've been thinking about the remote workshops also as a way to deliver SWC in places where they may struggle to get funds to cover instructor expenses (let alone the fee). It would be great to increase out outreach. Though in these cases, the local helpers may be possibly in low numbers and also will be unfamiliar with the method of teaching. 

things do get recorded! so it's not a disadvantage!

I think you have to have local room leaders and minimally tech-savvy TAs.
Or, at least, it's much much easier that way.

**Comment by chendaniely** [comment]
Thanks @ctb for the write up.

Tech was having intermittent packet drops.  Nothing too serious, just some words and parts of an explanation would get cut off.  I want to say enough of it was retained that we could fill in the blanks.

I almost considered seeing if we would benefit from spinning up a Jitsi/mumble server on digital ocean, but the rationale for using Hangouts makes sense.  If we want a backup streaming solution, I've used livecoding.tv to stream a git workshop when I cound'nt get hangouts on air working: https://www.livecoding.tv/video/2015-11-23-harvard-git/

Breaking up the responsibilities so that Emily was not the only one organizing and teaching seemed to make the entire experience seamless.  Although having gitter and etherpad seemed to be a little confusing for me on where I should be posting chat messages.  I had a separate laptop that had gitter and etherpad open, so I ended up typing in both sometimes.

The lesson material was very good and relevant.  The other students at Tech really enjoyed it.
I've went through Scott Murry's D3 [http://alignedleft.com/tutorials](tutorial) in the past, but the workshop had me leaving with the feeling that I can actually do something.  Maybe I'll circle back to Murry's book again, and fill in any blanks, but the workshop material was excellent.

**Comment by ctb** [comment]
Daniel, could you post this comment on

http://ivory.idyll.org/blog/2015-3hr-remote-workshops.html

pls? Alternatively I can do it ;)

thx,
--titus

On Mon, Dec 07, 2015 at 09:35:17AM -0800, Daniel Chen wrote:
> Thanks @ctb for the write up.
> 
> Tech was having intermittent packet drops.  Nothing too serious, just some words and parts of an explanation would get cut off.  I want to say enough of it was retained that we could fill in the blanks.
> 
> I almost considered seeing if we would benefit from spinning up a Jitsi/mumble server on digital ocean, but the rationale for using Hangouts makes sense.  If we want a backup streaming solution, I've used livecoding.tv to stream a git workshop when I cound'nt get hangouts on air working: https://www.livecoding.tv/video/2015-11-23-harvard-git/
> 
> Breaking up the responsibilities so that Emily was not the only one organizing and teaching seemed to make the entire experience seamless.  Although having gitter and etherpad seemed to be a little confusing for me on where I should be posting chat messages.  I had a separate laptop that had gitter and etherpad open, so I ended up typing in both sometimes.
> 
> The lesson material was very good and relevant.  The other students at Tech really enjoyed it.
> I've went through Scott Murry's D3 [http://alignedleft.com/tutorials](tutorial) in the past, but the workshop had me leaving with the feeling that I can actually do something.  Maybe I'll circle back to Murry's book again, and fill in any blanks, but the workshop material was excellent.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ctb/titus-blog/pull/12#issuecomment-162602037
-- 
C. Titus Brown, ctbrown@ucdavis.edu

**Pull request merged**